implementations/c/lib/memory/stdlib/tests/stdlib_memory_test.c

```diff
@@ -0,0 +1,157 @@
+#include <stdarg.h>
```

Comment:
This file looks great! One suggestion, since the test is calling the ockam_memory interface we should consider if this is a valid test for all implementations of ockam_memory or if its specific to stdlib.

I *think* this should apply to any implementation of ockam_memory so we might be better served if we moved this to something like compare.c and have it live in lib/memory/tests. Then this stdlib_memory_test.c file can call compare.c functions and as we grow the memory test suite other memory test functions.

Reply:
Good point, fixed that. Also, new version should allow to easily choose what implementations to test according to defines (e.g. different platform may have different implementations available)